draft-privacypass-rate-limit-tokens.md

```diff
@@ -243,11 +243,26 @@ Issuers MUST provide three parameters for configuration:
 1. Issuer Request URI: a token request URL for generating access tokens.
    For example, an Issuer URL might be https://issuer.example.net/token-request. This parameter
    uses resource media type "text/plain".
-1. Origin Name Key: a `KeyConfig` as defined in {{!OHTTP=I-D.thomson-http-oblivious}} to use when
-   encrypting the Origin Name in issuance requests. This parameter uses resource media type
-   "application/ohttp-keys".
+1. Origin Name Key: a `NameKey` structure as defined below to use when encrypting the Origin
+   Name in issuance requests. This parameter uses resource media type "application/pat-name-key".
```

Comment:
Can we not use "PAT" here, since it's not called private access tokens in this doc?

Reply:
Oh, sure, I didn't feel strongly about the name beyond that it should relate to privacy pass in some way.

Reply:
That's fine with me, yeah!

Reply:
Cool -- fixed!